sample/src/main/AndroidManifest.xml

```diff
@@ -29,7 +29,7 @@
       android:supportsRtl="true"
       android:theme="@style/Theme.TwilioVerifySnaAndroid">
     <activity
-        android:name="com.twilio.sample.MainActivity"
+        android:name="com.twilio.verify.sna.sample.MainActivity"
```

Comment:
should this also change https://github.com/twilio/twilio-verify-sna-android/pull/17/files#diff-c53f7bc57c4a7ffb92824cc2598ff253042b28a0d38f69eead012e62f7c41936R19?

Reply:
+1

Reply:
done